_posts/2019-10-08-unit-testing-basics.md

```diff
@@ -1117,15 +1117,15 @@ Most unit test frameworks will generate separate binaries for each `.cpp` unit
 test file written so that you can load them in a debugger (lldb or gdb).
 
 ```
-$ lldb build/kv_store/kv_store_tests
-(lldb) target create "build/kv_store/kv_store_tests"
-Current executable set to 'build/kv_store/kv_store_tests' (x86_64).
+$ lldb build/sum/sum_tests
```

Comment:
More straightforward for users continuing from minimal example

Reply:
<3